src/null-safety/index.md

```diff
@@ -91,8 +90,7 @@ Because pkg1 doesn't support null safety, version solving failed.
 The lower bound of "sdk: '>=2.9.0 <3.0.0'" must be 2.12.0 or higher to enable null safety.
 ```
 
-Libraries that opt out of null safety will cause analysis or compilation
-errors:
+Libraries incompatible with Dart 3 return analysis or compilation errors.
```

Comment:
This changes the meaning of the sentence. It's clear that libraries incompatible will cause errors, that's what incompatible means. The important part is the mention of the lack of null safety making them incompatible.

```suggestion
Pre-Dart 3 libraries that opt out of null safety will 
cause analysis or compilation errors.
```